sdk/identity/azure-identity/pom.xml

```diff
@@ -27,7 +27,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core</artifactId>
-      <version>1.5.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
+      <version>1.6.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core;current} -->
```

Comment:
`core` is not in `identity` group. So, it should use `dependency` tag. If you want to take a dependency on unreleased version of core, then you should use `<!-- {x-version-update;unrelease_com.azure:azure-core;dependency} -->`. Also, add the unreleased version to [version_client.txt](https://github.com/Azure/azure-sdk-for-java/blob/master/eng/versioning/version_client.txt#L61)

Reply:
Thanks! fixed.